app/views/laundry_rooms/index.html.erb

```diff
@@ -8,6 +8,7 @@
               <% if booking[:time] > (DateTime.now - 5.hours) %>
                 <ul>
                   <li><%= booking[:time].beginning_of_hour.to_formatted_s(:short) %></li>
+                  <%= link_to 'Unbook', laundry_room_delete_booking_path(laundry_room_id: @laundry_rooms.ids, time: booking[:time]) %>
```

Comment:
Why `@laundry_rooms.ids`?  Why do you want to pass in several ids here? (I did not spot that in my first review, sorry)

Reply:
Yes. Because we have set the `@laundry_rooms = LaundryRoom.all`. If we we're to call on only one laundry room, the only solutions I can think of right now is: 
1. Set `@laundry_room = LaundryRoom.first`, and then change A LOT of code
2. Hard code the laundry_room id to the first laundry room.

Reply:
Magnus suggested "we could change the `@laundry_rooms.ids` to `@laundry_rooms.first`  and its only that change we need to do" so I will do that.